shared/js/notification_helper.js

```diff
@@ -30,26 +30,29 @@
     // an object -> {raw: string}
     send: function nh_send(titleL10n, options) {
       return new Promise(function(resolve, reject) {
```

Comment:
`formatValue` already returns a promise, so there's no need to create a new one here and resolve it manually.